pydocx/export/base.py

```diff
@@ -472,7 +477,8 @@ def export_numbering_item(self, numbering_item):
         return self.yield_nested(numbering_item.children, self.export_node)
 
     def export_simple_field(self, simple_field):
-        default_results = self.yield_nested(simple_field.children, self.export_node)
+        default_results = self.yield_nested(
+            simple_field.children, self.export_node)
```

Comment:
I have pep8 1.5.7 installed too. I also have flake8 2.4.1, which is what travis installs each time the tests are run. flake8 reads the `[flake8]` section in `tox.ini` which has `max-line-length = 95`.